aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -110,6 +129,12 @@ module aptos_framework::account {
         });
     }
 
+    public(friend) fun create_address_map(aptos_framework_account: &signer) {
+        move_to(aptos_framework_account, OriginatingAddress {
```

Comment:
this table is the size of the all accounts that rotated key once. It is very large and stored at 0x1 and may have performance concerns cc:  @msmouse for insights?

Reply:
there's no performance impact based upon where a table is stored, it is decoupled just like any account is decoupled from another.

Reply:
> [#566 (comment)](https://github.com/aptos-labs/aptos-core/issues/566#issuecomment-1176642851) @areshand feel free to read the above.
After reading the long discussion of thread, I am not sure if there is agreement of using on-chain storage over indexer events.

I don't understand the conclusion `the lack of canonical and verifiable indexes and events is concerning.` 
The verification is through auth key and address. If an indexer lies or event is wrong, the auth_key just cannot sign for an address. 

Meanwhile, we pay a huge cost for storing these on-chain since this could be really large. since all the data are stored under 0x1 account, we are not concerned by one particular account with much large data size compared to other account?

Reply:
Ok we don't have a verification problem then. In terms of data availability, that is the indexer's problem.people can build indexer by storing all the data to make the data available. Not sure why the chain or framework wants to provide a guarantee on remembering all the original addresses for an auth_key. This is more like a service instead of core functionality of a blockchain.